UnityProject/Assets/Scripts/NPC/AI/Hostile/FaceHugAction.cs

```diff
@@ -67,12 +72,17 @@ private void Facehug(Equipment playerInventory, RegisterPlayer player)
 			var result = Spawn.ServerPrefab(maskObject);
 			var mask = result.GameObject;
 
-			Inventory.ServerAdd(
-				mask,
-				playerInventory.ItemStorage.GetNamedItemSlot(NamedSlot.mask),
-				ReplacementStrategy.DespawnOther);
+			foreach (var itemSlot in playerInventory.ItemStorage.GetNamedItemSlots(NamedSlot.mask))
+			{
+				Inventory.ServerAdd(
+					mask,
+					itemSlot,
+					ReplacementStrategy.DespawnOther);
+				break;
+			}
 
-			_ = Despawn.ServerSingle(gameObject);
+
+			Despawn.ServerSingle(gameObject);
```

Comment:
```suggestion
			_ = Despawn.ServerSingle(gameObject);
```
To silence compiler warnings.